CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# these owners will be requested for
+# review when someone opens a pull request.
+*       @NikoHadouken @volkramweber @steffenkleinle @Stift
```

Comment:
@Stift I am a little confused if you are the "right" Christian and whether you will work on the flutter app? :D 
If you are, do you want to get review notifications?
If you are not, could you perhaps answer with the correct Github handle of the other Christian? Thank you :)